### ISSUE TITLE
Use term DAO instead of DAE

### ISSUE THREAD

**Issue by xaur** [xaur/decred-issues#134]
Discussions:

* [2019-06-07](https://matrix.to/#/!lbzTjhzNbIaDbuAxkS:decred.org/$15599369468636TaKWs:decred.org): shall we use DAO instead of DAE?; DAE was used by Ditto when talking to reporters; DAOs are hot and we're missing out, it's also a SEO issue; we have to explain why we don't call it a DAO
* [2019-06-13](https://www.reddit.com/r/decred/comments/c02geh/poll_rename_dae_to_dao/): days of association with The DAO are gone; Decred is missing all the "DAO" rage and is absent from many conversations; several community members explain "DAE" as our version of "DAO"

**Comment by nottrunner** [comment]
DAE could be confusing and requires explanation. I'd be for the industry standard term - DAO.  Its the same thing, I don't see much positive value in calling ourselves a DAE.

**Comment by noahpierau** [comment]
These articles could be changed:

- https://medium.com/@richardred/working-for-the-decred-dae-a9cfb17686fa
- https://medium.com/@seth.benton/how-i-make-a-living-in-the-crypto-gig-economy-8674f3693858
- https://medium.com/@changhugo/decred-as-a-dae-infrastructure-provider-970677f38179

**Comment by denyszayets** [comment]
Changed to DAO in Italian and Russian articles.

**Comment by xaur** [comment]
Looks like decent progress was made. Shall we close this or is there more work? cc @s-ben

**Comment by s-ben** [comment]
@xaur, it looks like this change is "official", and the docs have already been changed. However, decred.org still has a couple uses of DAE in the [roadmap](https://decred.org/roadmap/) (PR to change this has been [merged](https://github.com/decred/dcrweb/pull/681), but not seeing it deployed live yet (can you??)). 

Also, I still think there is a little work to do getting the word out internally (June's DJ should help), potentially editing Medium pieces that reference the DAE, perhaps some internal comms (was going to ask #Marketing if that's a good idea or better to just update quietly). So maybe keep this open until those lingering issues are closed?

**Comment by xaur** [comment]
Good point, let's keep it open for a while. And no I can't deploy to decred.org.

**Comment by oiezz** [comment]
@RichardRed0x these articles could be changed as well: 
- https://medium.com/decred/on-the-launch-of-decreds-politeia-b305d06d8bda
- https://medium.com/decred/governance-of-the-decred-project-subsidy-through-politeia-e6393b2bab1f

**Comment by s-ben** [comment]
This made me realize that I needed to change my [medium piece ](https://medium.com/@seth.benton/how-i-make-a-living-in-the-crypto-gig-economy-8674f3693858) as well. Spent a few minutes Googling for possible negative consequences of editing a medium piece, didn't find any. Was able to edit my piece fairly easily, and still seeing all my stats in tact...

@RichardRed0x is a bit of a journo, so curious if he wants to edit his piece on working for the DAE or not,
https://medium.com/@richardred/working-for-the-decred-dae-a9cfb17686fa

There is also this article by @changhugo (on medium, don't know his GitHub name), 
https://medium.com/@changhugo/decred-as-a-dae-infrastructure-provider-970677f38179

**Comment by changhugo** [comment]
@s-ben  Thanks for the note, I have updated my article.

**Comment by xaur** [comment]
A good practice is to add an "Update: " notice with short summary of the edit.
